dist/components/Node.js

```diff
@@ -56,11 +56,11 @@ var Node = function (_React$Component) {
   (0, _createClass3.default)(Node, [{
     key: 'render',
     value: function render() {
-      var _props = this.props;
-      var node = _props.node;
-      var depth = _props.depth;
-      var tagStyle = stylesheet.tagStyle;
-      var containerStyle = stylesheet.containerStyle;
+      var _props = this.props,
+          node = _props.node,
```

Comment:
This is actually being built by the build tool.  I can take a look a little deeper at the build process for this if you want.  Or another alternative it to ignore the dist directory and have a post install script to build the dist when people install the package?